src/Dialect/Mlir/DialectBuilder.cpp

```diff
@@ -653,6 +654,20 @@ void SCFBuilder::ifThenElse(Value cond,
   }
 }
 
+void SCFBuilder::parallelLoop(ValueRange lowerBounds, ValueRange upperBounds,
+    ValueRange steps,
+    function_ref<void(DialectBuilder &createKrnl, ValueRange)> bodyFn) const {
+  // SmallVectorImpl<Value> ivStorage;
+  b.create<scf::ParallelOp>(loc, lowerBounds, upperBounds, steps,
+      [&](OpBuilder &childBuilder, Location childLoc,
+          ValueRange inductionVars) {
+        KrnlBuilder builder(childBuilder, childLoc);
+        printf("%d\n", inductionVars.size());
```

Comment:
is this leftover "printf debugging" that should be removed? I happened to notice it because it causes the warning:
```
src/Dialect/Mlir/DialectBuilder.cpp:665:24: warning: format specifies type 'int' but the argument has type 'size_t' (aka 'unsigned long') [-Wformat]
        printf("%d\n", inductionVars.size());
```

Reply:
I'm removing it in PR #1644

Reply:
Thanks